views/index.erb

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="//fonts.googleapis.com/css?family=font1|font2|etc"
+      type="text/css"
+    />
+    <link rel="stylesheet" href="/style.css" type="text/css" />
+  </head>
+  <body>
+    <div class="container">
+    <div class="header">
+    <h1>Chitter</h1>
+    </div>
+      <div class="messages">
+        <% @messages.each do |message| %>
+          <div class="message">
+          <h3><%= message.name %></h3>
+          <h4>@<%= message.username %></h4>
+          <p>
+          <%= message.time.day %>/<%= message.time.month %>/<%= message.time.year%> 
+          <%=message.time.hour%>:<%= message.time.min%>
+          </p>
```

Comment:
It might be dryer to try to make time format defined in message.rb like "%Y-%m-%d %H:%M:%S", so will be able to call just "time" here